bundles/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/weather/NetatmoWeatherBinding.java

```diff
@@ -180,13 +180,19 @@ public void execute(OAuthCredentials oauthCredentials, Collection<NetatmoBinding
                                 }
                             }
                                 break;
+                            case BATTERYPERCENT:
+                            case BATTERYSTATUS:
                             case BATTERYVP:
                             case RFSTATUS:
                                 for (Device device : stationsDataResponse.getDevices()) {
                                     for (Module module : device.getModules()) {
                                         if (module.getId().equals(moduleId)) {
                                             switch (measureType) {
+                                                case BATTERYPERCENT:
                                                 case BATTERYVP:
```

Comment:
yes that is correct. This is from the Netatmo documentation:

    Number   Netatmo_Outdoor_Batteryvp "Outdoor battery status [%d %%]"  (Netatmo {netatmo="00:00:00:00:00:00#02:00:00:00:00:00#Batteryvp"}